### PR TITLE
change how we do requirements and bump version

### PR DESCRIPTION
**Pull request by jspiro**
So that virtualenv can build the deps.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,15 +10,19 @@ def read(fname):
...
-    name = "pystatsd",
-    version = "0.0.1",
-    author = "John Koenig",
-    author_email = "john@postmates.com",
-    description = ("Thread safe statsd API"),
-    license = "MIT",
-    keywords = "python statsd",
-    url = "https://github.com/postmates/pystatsd",
+    name="pystatsd",
+    version="1.0.0",
```
Comment (pulltab) [inline setup.py]: You are going to introduce the 1.0.0 tag?

File: Makefile [hunk 2]
```diff
@@ -24,7 +24,3 @@ test: check
 .PHONY: check
 check:
 	python -m unittest discover
-
-.PHONY: deps
-deps:
-	pip install -r requirements.txt
```
Comment (pulltab) [inline Makefile]: Why not use pip?

File: setup.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import os
 from setuptools import setup
 
+
```
Comment (pulltab) [inline setup.py]: Extra whitespace.

Comment (jspiro) [inline setup.py]: Actually this is flake8 style

File: Makefile (hunk 2 above)
Comment (jspiro) [inline Makefile]: Because setup tools won't install it

File: setup.py (hunk 1 above)
Comment (jspiro) [inline setup.py]: Why not, it's the first version :)

Comment (pulltab) [inline setup.py]: No reason.  Just making sure you were going to push the tag. :D

**Comment by pulltab** [comment]
LGTM

**Pull request merged**